mitosheet/src/mito/components/taskpanes/Graph/graphUtils.tsx

```diff
@@ -238,20 +186,36 @@ export const getGraphTypeFullName = (graphType: GraphType): string => {
     }
 }
 
+export const convertBackendtoFrontendGraphData = (graphDataBackend: GraphDataBackend): GraphDataFrontend => {
+    return {
+        graphID: graphDataBackend.graph_id,
+        graphOutput: graphDataBackend.graph_output,
+        graphTabName: graphDataBackend.graph_tab_name,
+    }
+}
+
+export const convertFrontendtoBackendGraphData = (graphDataFrontend: GraphDataFrontend): GraphDataBackend => {
+    return {
+        graph_id: graphDataFrontend.graphID,
+        graph_output: graphDataFrontend.graphOutput,
+        graph_tab_name: graphDataFrontend.graphTabName,
+    }
+}
```

Comment:
This feels like overkill. Let's just use the python variables on the frontend too - and skip this conversion.